PubNub/Data/Service Objects/PNResult.h

```diff
@@ -73,6 +73,8 @@
  */
 @property (nonatomic, readonly, copy) NSURLRequest *clientRequest;
 
+//@property (nonatomic, readonly, strong) NSNumber *region;
+
```

Comment:
@jzucker2 Is this code alive? If debug information from message envelop should be saved somewhere, I suggest it to be some separate class like in 3.x version.